### ISSUE TITLE
using data augmentation decreases the resolution

### ISSUE THREAD

**Issue by AyaMohamedS** [hellochick/PSPNet_tensorflow#53]
i trained the downloaded pspnet model 1 epoch over all the ade20k training dataset and i got miou=0.274 because of the small batch size used (batchsize=1)
but when i used random scale and mirror during training the accuracy decreased more.

what is the reason ? and what is the best no. of epochs overall the dataset to show the effect of using data augmentation ?
